ResearchKit/Consent/ORKVisualConsentStepViewController.m

```diff
@@ -506,10 +512,24 @@ - (void)observedScrollViewDidScroll:(UIScrollView *)scrollView {
 }
 
 - (void)showViewController:(ORKConsentSceneViewController *)viewController forward:(BOOL)forward animated:(BOOL)animated {
+    [self showViewController:viewController forward:forward animated:animated preloadNextViewController:YES];
+}
+
+- (void)showViewController:(ORKConsentSceneViewController *)viewController forward:(BOOL)forward animated:(BOOL)animated preloadNextViewController:(BOOL)preloadNextViewController {
     [self showViewController:viewController
                      forward:forward
                     animated:animated
-                  completion:nil];
+                  completion:^(BOOL finished) {
+                      if (preloadNextViewController) {
+                          ORKConsentSceneViewController *nextViewController = [self viewControllerForIndex:[self currentIndex]+1];
```

Comment:
I did this thinking on a compact way of handling both the default active consent images, and any user-supplied custom ones. *GlucoSuccess*, for example, uses custom images together with its own custom animations.